test/spec.yml

```diff
@@ -101,7 +101,7 @@
         [201*-*-*T*:*:*Z] Function invocation took * us
     - in: bn list --json
       out: |-
-        *[*{"name":"rand*","lastDeployed":"[20*-*-*T*:*:*Z]"}*]*
+        *[*{"name":"rand*","lastDeployed":"20*-*-*T*:*:*Z"}*]*
```

Comment:
Use `%` instead of `*`, especially for long outputs like `bn list`.  I added this to avoid #393 and similar.

Reply:
done in d78d791